cmd/troubleshoot/dynakube_test.go

```diff
@@ -8,7 +8,6 @@ import (
 	"github.com/Dynatrace/dynatrace-operator/pkg/api/shared/value"
```

Comment:
```
Volumes:                                                                                                                                                                                                       │
│   host-root:                                                                                                                                                                                                   │
│     Type:          HostPath (bare host directory volume)                                                                                                                                                       │
│     Path:          /                                                                                                                                                                                           │
│     HostPathType:                                                                                                                                                                                              │
│   connection-info-secret:                                                                                                                                                                                      │
│     Type:        Secret (a volume populated by a Secret)                                                                                                                                                       │
│     SecretName:  dynakube-test-oneagent-tenant-secret                                                                                                                                                          │
│     Optional:    false    
``` 

These are the volumes I see in the oneAgent pods when running this scenario:

```
  oneAgent:
    hostMonitoring: 
      env:
          - name: ONEAGENT_ENABLE_VOLUME_STORAGE # needed on GKE
            value: "true"
```

Shouldn't there be an emptyDir? Am I missing something?

Reply:
no, `ONEAGENT_ENABLE_VOLUME_STORAGE ` will cause the agent do it's container magic AFAIK

Reply:
Then pls update the review section as you wrote there that there should be an emptyDir in this scenario

Otherwise I tested everything and it LGTM 👍